tests/infra/consortium.py

```diff
@@ -535,9 +647,9 @@ def set_node_certificate_validity(
     ):
         proposal_body, careful_vote = self.make_proposal(
             "set_node_certificate_validity",
-            node_to_renew.node_id,
-            valid_from,
-            validity_period_days,
+            node_id=node_to_renew.node_id,
+            valid_from=valid_from,
+            validity_period_days=validity_period_days,
```

Comment:
Just to make sure this change is clear:

Previously this passed some `*args` through a looked-up-by-name function to `proposal_generator`, where they were type-checked/converted to the correct types, for embedding into JSON. Now the names and types defined here are directly inserted into the JSON object, so the values here need to match the expectations in the corresponding JS action.